### ISSUE TITLE
Generating the augmented dataset

### ISSUE THREAD

**Issue by dadsteggun** [faraway1nspace/PCRD_JAGS_demo#2]
I think this only works when n.aug <= N: 
(line 125 of R_PCRD_JAGS_hierarchical.R)

`Y.aug[(N+1):mm,,]<- 0 * Y.tt[1:n.aug,,] `

But this is just to fill in zeroes, so why not just do

`Y.aug[(N+1):mm,,]<- 0` ?

**Comment by faraway1nspace** [comment]
Good point. That line was intended to maintain the NA structure in the 3D array of captures. I'll have to rethink this for `n.aug < N`

**Comment by faraway1nspace** [comment]
Alright should be fixed. Have a go a let me know

**Comment by dadsteggun** [comment]
I don't quite follow the edit but perhaps I'm missing something. The NA structure in the Y.tt's should be fine since the index `Y.aug[(N+1):mm,,]` already explicitly excludes the Y.tt's. I just tested with NA's in the Y.tt's and `Y.aug[(N+1):mm,,]<- 0` works fine.
